diff --git a/core/java/android/util/Log.java b/core/java/android/util/Log.java
index 0965a8c59234..351a73d7195b 100644
--- a/core/java/android/util/Log.java
+++ b/core/java/android/util/Log.java
@@ -147,7 +147,8 @@ public final class Log {
      * @return A positive value if the message was loggable (see {@link #isLoggable}).
      */
     public static int v(@Nullable String tag, @NonNull String msg) {
-        return println_native(LOG_ID_MAIN, VERBOSE, tag, msg);
+        return 0;
+        //return println_native(LOG_ID_MAIN, VERBOSE, tag, msg);
     }
 
     /**
@@ -170,7 +171,8 @@ public final class Log {
      * @return A positive value if the message was loggable (see {@link #isLoggable}).
      */
     public static int d(@Nullable String tag, @NonNull String msg) {
-        return println_native(LOG_ID_MAIN, DEBUG, tag, msg);
+        return 0;
+        //return println_native(LOG_ID_MAIN, DEBUG, tag, msg);
     }
 
     /**
@@ -193,7 +195,8 @@ public final class Log {
      * @return A positive value if the message was loggable (see {@link #isLoggable}).
      */
     public static int i(@Nullable String tag, @NonNull String msg) {
-        return println_native(LOG_ID_MAIN, INFO, tag, msg);
+        return 0;
+        //return println_native(LOG_ID_MAIN, INFO, tag, msg);
     }
 
     /**
@@ -466,6 +469,9 @@ public final class Log {
      */
     public static int printlns(int bufID, int priority, @Nullable String tag, @NonNull String msg,
             @Nullable Throwable tr) {
+        if (priority <= INFO) {
+            return 0;
+        }
         ImmediateLogWriter logWriter = new ImmediateLogWriter(bufID, priority, tag);
         // Acceptable buffer size. Get the native buffer size, subtract two zero terminators,
         // and the length of the tag.
diff --git a/core/jni/android_util_Log.cpp b/core/jni/android_util_Log.cpp
index ad35b7d39539..8f19b7b3e67c 100644
--- a/core/jni/android_util_Log.cpp
+++ b/core/jni/android_util_Log.cpp
@@ -77,6 +77,10 @@ static jint android_util_Log_println_native(JNIEnv* env, jobject clazz,
     const char* tag = NULL;
     const char* msg = NULL;
 
+    if (priority <= ANDROID_LOG_INFO) {
+        return 0;
+    }
+
     if (msgObj == NULL) {
         jniThrowNullPointerException(env, "println needs a message");
         return -1;
