diff --git a/logging.cpp b/logging.cpp
index 07ba382..87780bd 100644
--- a/logging.cpp
+++ b/logging.cpp
@@ -232,7 +232,7 @@ void SetDefaultTag(const std::string& tag) {
 static bool gInitialized = false;
 
 // Only used for Q fallback.
-static LogSeverity gMinimumLogSeverity = INFO;
+static LogSeverity gMinimumLogSeverity = WARNING;
 
 #if defined(__linux__)
 static void KernelLogLine(const char* msg, int length, android::base::LogSeverity severity,
@@ -524,6 +524,9 @@ std::ostream& LogMessage::stream() {
 void LogMessage::LogLine(const char* file, unsigned int line, LogSeverity severity, const char* tag,
                          const char* message) {
   int32_t priority = LogSeverityToPriority(severity);
+  if (priority < ANDROID_LOG_WARN) {
+      return;
+  }
   if (__builtin_available(android 30, *)) {
     __android_log_message log_message = {
         sizeof(__android_log_message), LOG_ID_DEFAULT, priority, tag, file, line, message};
@@ -556,6 +559,9 @@ bool ShouldLog(LogSeverity severity, const char* tag) {
   // take into consideration the value from SetMinimumLogSeverity().
   if (__builtin_available(android 30, *)) {
     int32_t priority = LogSeverityToPriority(severity);
+    if (priority < ANDROID_LOG_WARN) {
+      return false;
+    }
     return __android_log_is_loggable(priority, tag, ANDROID_LOG_INFO);
   } else {
     return severity >= gMinimumLogSeverity;
