diff --git a/aosp/hardware_android.cc b/aosp/hardware_android.cc
index 61b23886..0052626b 100644
--- a/aosp/hardware_android.cc
+++ b/aosp/hardware_android.cc
@@ -265,8 +265,7 @@ int64_t HardwareAndroid::GetBuildTimestamp() const {
 // Returns true if the device runs an userdebug build, and explicitly allows OTA
 // downgrade.
 bool HardwareAndroid::AllowDowngrade() const {
-  return GetBoolProperty("ro.ota.allow_downgrade", false) &&
-         GetProperty("ro.build.type", "") != "user";
+  return GetBoolProperty("ro.ota.allow_downgrade", false);
 }
 
 bool HardwareAndroid::GetFirstActiveOmahaPingSent() const {
diff --git a/payload_consumer/postinstall_runner_action.cc b/payload_consumer/postinstall_runner_action.cc
index 23f16e16..f4effe34 100644
--- a/payload_consumer/postinstall_runner_action.cc
+++ b/payload_consumer/postinstall_runner_action.cc
@@ -235,10 +235,10 @@ bool PostinstallRunnerAction::MountPartition(
     }
 
     // Run backuptool script
-    int ret = system("/postinstall/system/bin/backuptool_postinstall.sh");
-    if (ret == -1 || WEXITSTATUS(ret) != 0) {
-      LOG(ERROR) << "Backuptool postinstall step failed. ret=" << ret;
-    }
+    // int ret = system("/postinstall/system/bin/backuptool_postinstall.sh");
+    // if (ret == -1 || WEXITSTATUS(ret) != 0) {
+    //   LOG(ERROR) << "Backuptool postinstall step failed. ret=" << ret;
+    // }
 
     // Switch back to update_engine domain
     if (setexeccon(nullptr)) {
