diff --git a/AssembleVintf.cpp b/AssembleVintf.cpp
index 34ffd10..1d5b142 100644
--- a/AssembleVintf.cpp
+++ b/AssembleVintf.cpp
@@ -334,7 +334,7 @@ class AssembleVintfImpl : public AssembleVintf {
         out() << "    Input:" << std::endl;
         for (const auto& e : inputs) {
             if (!e.fileName().empty()) {
-                out() << "        " << e.fileName() << std::endl;
+                out() << "        " << std::endl;
             }
         }
         out() << "-->" << std::endl;
