diff --git a/build/soong/mac_permissions.go b/build/soong/mac_permissions.go
index 1df8a51ff..7cf880240 100644
--- a/build/soong/mac_permissions.go
+++ b/build/soong/mac_permissions.go
@@ -103,7 +103,7 @@ func (m *macPermissionsModule) GenerateAndroidBuildActions(ctx android.ModuleCon
 	rule.SandboxDisabled()
 	rule.Command().
 		Tool(ctx.Config().PrebuiltBuildTool(ctx, "m4")).
-		Text("--fatal-warnings -s").
+		Text("--fatal-warnings").
 		FlagForEachArg("-D", ctx.DeviceConfig().SepolicyM4Defs()).
 		Inputs(keys).
 		FlagWithOutput("> ", m4Keys).
diff --git a/build/soong/policy.go b/build/soong/policy.go
index 97ac07979..a2840e56b 100644
--- a/build/soong/policy.go
+++ b/build/soong/policy.go
@@ -293,7 +293,6 @@ func (c *policyConf) transformPolicyToConf(ctx android.ModuleContext) android.Ou
 		FlagWithArg("-D target_restricts_ashmem_usage=", c.restrictAshmemUsage(ctx)).
 		Flag(boardApiLevelToM4Macro(ctx, c.properties.Board_api_level)).
 		Flags(flagsToM4Macros(flags)).
-		Flag("-s").
 		Inputs(srcsWithNewline).
 		Text("> ").Output(conf)
 
diff --git a/build/soong/selinux_contexts.go b/build/soong/selinux_contexts.go
index ffaff09e9..856444291 100644
--- a/build/soong/selinux_contexts.go
+++ b/build/soong/selinux_contexts.go
@@ -297,7 +297,7 @@ func (m *selinuxContextsModule) buildGeneralContexts(ctx android.ModuleContext,
 	flags := m.getBuildFlags(ctx)
 	rule.Command().
 		Tool(ctx.Config().PrebuiltBuildTool(ctx, "m4")).
-		Text("--fatal-warnings -s").
+		Text("--fatal-warnings").
 		FlagForEachArg("-D", ctx.DeviceConfig().SepolicyM4Defs()).
 		Flag(boardApiLevelToM4Macro(ctx, m.properties.Board_api_level)).
 		Flags(flagsToM4Macros(flags)).
